css-view-transitions-2/Overview.bs

```diff
@@ -0,0 +1,400 @@
+		In the <a href="https://html.spec.whatwg.org/#script-processing-model">Script processing model</a> section,
+		move <a href="https://html.spec.whatwg.org/#script-processing-model:unblock-rendering">unblock rendering</a> step to the end.
+
+		Note: this ensures that the [=reveal document=] steps are called after firing script `load` or `error` events.
+	</div>
+
+	<div algorithm="monkey patch to reactivation">
+		Run the following step in [=Document/reactivate=], before step 4 (querying for [=page showing=]):
+
+		1. If |document| is not [=render-blocked=], then [=reveal document|reveal=] |document|.
+	</div>
+
+	<div algorithm="page reveal">
+	To <dfn>reveal {{Document}}</dfn> |document|:
+		1. Assert: |document|'s [=page showing=] is false.
+
+		1. Fire a new event named <code>reveal</code> on |document|'s [=relevant global object=],
+			using {{PageRevealEvent}}.
+
+		1. Let |transition| be the result of getting the [=inbound cross-document view-transition=] for |document|.
+
+		1. If |transition| is not null, then [=activate view transition|activate=] |transition|.
+	</div>
+
+## Setting up and activating the cross-document view transition ##
+
+	<div algorithm>
+		To <dfn>setup outbound cross-document view transition</dfn> given a {{Document}} |oldDocument|,
+		a {{Document}} |newDocument|, and |onReady|, which is an algorithm accepting nothing:
+
+		1. If |oldDocument|'s [=environment settings object/origin=] is not [=same origin=] as
+			|newDocument|'s [=environment settings object/origin=] then call |onReady| and return.
+
+		1. If |newDocument| [=was created via cross-origin redirects=] is true
+			and |newDocument|'s [=latest entry=] is null, then call |onReady| and return.
+
+			Note: A document with a non-null [=latest entry=]
+			is being [=Document/reactivated=], in which case we don't need to check for cross-origin redirects.
+
+		1. If |oldDocument| does not [=opt in to cross-document view transitions=], then call |onReady| and return.
+
+			Note: We don't know yet if |newDocument| has opted in, as it might not be parsed yet.
+			We check the opt-in for |newDocument| when it is [=reveal document|revealed=].
+
+		1. If |oldDocument|'s [=active view transition=] is not null,
+			then [=skip the view transition|skip=] |oldDocument|'s [=active view transition=]
+			with an "{{AbortError}}" {{DOMException}} in |oldDocument|'s [=relevant Realm=].
+
+			Note: this means that any running transition would be skipped when the document is ready
+			to unload.
+
+		1. Let |outboundTransition| be a new {{ViewTransition}} object in |oldDocument|'s [=relevant Realm=],
+			whose [=ViewTransition/process old state captured=] is set to the following steps:
+
+			1. |outboundTransition|'s [=ViewTransition/phase=] is not "`pending-capture`", then call |onReady| and return.
+
+			1. If |oldDocument| does not [=opt in to cross-document view transitions|opts in to cross-document view transitions=],
```

Comment:
Why do we need a check here? Didn't we already do it before initiating the capture?